### PR TITLE
PMM-10349 DBaaS easy access to deploy button

### PR DESCRIPTION
**Pull request by solovevayaroslavna**
https://jira.percona.com/browse/PMM-10349
To deploy the initial DB Cluster, the user who just configured K8s needs to know about the second tab in UI to test the deployment.  It will be better to give them an easy solution to deploy into just added k8s Cluster 

[Запись встречи 17.08.2022 17-06-45 - запись.webm](https://user-images.githubusercontent.com/90199600/185141991-e2682077-da4d-4300-b2c6-29b61e8a7f2a.webm)

### CODE REVIEW

File: public/app/percona/dbaas/components/DBCluster/DBCluster.tsx [hunk 1]
```diff
@@ -37,12 +47,13 @@ import { logger } from '@sentry/utils';
...
 export const DBCluster: FC = () => {
   const styles = useStyles(getStyles);
-  const [addModalVisible, setAddModalVisible] = useState(false);
   const [deleteModalVisible, setDeleteModalVisible] = useState(false);
   const [editModalVisible, setEditModalVisible] = useState(false);
   const [logsModalVisible, setLogsModalVisible] = useState(false);
   const [updateModalVisible, setUpdateModalVisible] = useState(false);
   const [selectedCluster, setSelectedCluster] = useState<Cluster>();
+  const { selectedKubernetesCluster } = useSelector(getDBaaS);
+  const [addModalVisible, setAddModalVisible] = useState(selectedKubernetesCluster ? true : false);
```
Comment (fabio-silva) [inline public/app/percona/dbaas/components/DBCluster/DBCluster.tsx:56]:
```suggestion
  const [addModalVisible, setAddModalVisible] = useState(selectedKubernetesCluster ? true : false);
```
```suggestion
  const [addModalVisible, setAddModalVisible] = useState(!!selectedKubernetesCluster);
```

**Review by fabio-silva — CHANGES_REQUESTED** [review]
Good job @solovevayaroslavna 🙌 
I just have one question: we're relying on `selectedKubernetesCluster` to know if the user selected something. Shouldn't we set it to null afterwards? If we never clean it from the store, the modal will open with those values everytime we go to the page. Am I seeing this right?

**Comment by fabio-silva** [comment]
P.S. Sorry, I just realised you already set it to null on unmount, good one!

File: public/app/percona/dbaas/components/DBCluster/DBCluster.tsx (hunk 1 above)
Comment (solovevayaroslavna) [reply public/app/percona/dbaas/components/DBCluster/DBCluster.tsx:56]: @fabio-silva Yes, if the user has selected a cluster, then we open the model with the default values of this cluster (previously we opened it with the default value of the first cluster from the list)

In the dbclusters component (dbclusters page) I added useEffect with return function where I reset to null (i.e. as soon as the user leaves this page, the value will be reset to null and the first cluster from the list will be shown in the open modal again by default)

**Comment by solovevayaroslavna** [comment]
@fabio-silva I corrected according to your remark with !!

**Review by matejkubinec — DISMISSED** [review]

**Review by fabio-silva — DISMISSED** [review]

**Review by fabio-silva — DISMISSED** [review]

**Review by matejkubinec — APPROVED** [review]

**Pull request merged**